UIKitDZ/UIKitDZ/Storage.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  Storage.swift
+//  UIKitDZ
+//
+//  Created by Анна Сычева on 03.10.2022.
+//
+
+import UIKit
+
+// MARK: - SaveLoginPass
+/// Хранение и проверка логина и пароля
+final class SaveLoginPass {
+    
+    // MARK: - Public properties
+    
+    static var shared = SaveLoginPass()
+    
+    public lazy var defaults = UserDefaults.standard
```

Comment:
```suggestion
    lazy var defaults = UserDefaults.standard
```
удаляем везде public